tasks/ilo.yml

```diff
@@ -1,34 +1,33 @@
 ---
 # vim:ft=ansible:
 
- - assert: { that: "ansible_system_vendor == 'HP'" }
+- assert: { that: "ansible_system_vendor == 'HP'" }
 
-   # We look for:
-   # 01:00.0 "System peripheral" "Hewlett-Packard Company" "Integrated Lights-Out Standard Slave Instrumentation & System Support" -r05 "Hewlett-Packard Company" "iLO4"
- - name: Checking for ILO device
-   command: lspci -s 01:00.0 -m
-   register: ilo_info
-   changed_when: False
-   check_mode: no
+- name: Checking for ILO device
+  shell: hponcfg -w conf.txt|grep -i 'Firmware Revision'| awk '{print $8$9}'
```

Comment:
Turns out our company has some HPE Gen10 servers! 

The output from the lspci command is this:

<pre>
01:00.0 "System peripheral" "Hewlett-Packard Company" "Integrated Lights-Out Standard Slave Instrumentation & System Support" -r07 "Hewlett Packard Enterprise" "iLO5"`
</pre>

Which based on my awesome bash hacking should output "iLO5"

<pre>
$ echo '01:00.0 "System peripheral" "Hewlett-Packard Company" "Integrated Lights-Out Standard Slave Instrumentation & System Support" -r06 "Hewlett-Packard Company" "iLO4"'|rev|cut -d '"' -f2|rev
iLO5
</pre>

As long as the lspci looks like the above then we should be OK with the existing.